src/Appwrite/Utopia/Response.php

```diff
@@ -257,8 +258,7 @@ public function dynamic(Document $document, string $model): void
             $item = self::getFilter()->parse($output, $model);
         }
 
-        $this->json($output);
-
+        $this->json(!empty($output) ? $output : new stdClass());
```

Comment:
Can we not do the following here 
```suggestion
        $this->json($output ?? new stdClass());
```

Reply:
I wasn't 100% sure so I doubled check, and the output is not the same for both lines. ?? is similar to `isset` and check if the assoc array has a specific key. Empty will also return true for empty string or arrays.

This is the code I used for the test:

```php
$output = [];

echo "\n";

echo '1:'.json_encode(!empty($output) ? $output : new \stdClass());

echo "\n";

echo '2:'.json_encode($output ?? new \stdClass());
```

results:
```php
// 1:{}
// 2:[]
```

Reply:
okay eldad 👍